README.md

```diff
@@ -36,7 +36,7 @@ Why do we need yet another serialization format? Borsh is the first serializer t
    * Consistent means there is a bijective mapping between objects and their binary representations. There is no two binary representations that deserialize
    into the same object. This is extremely useful for applications that use binary representation to compute hash;
    * Borsh comes with a full specification that can be used for implementations in other languages;
-* Safe. Borsh implementations use safe coding practices. In Rust, Borsh uses only safe code;
```

Comment:
Also the website, please.

Reply:
See `docs`.

Reply:
Thanks. just updated, is website auto deployed?

Reply:
sorry seems it's auto reformat on save by editor, i'll change it back now

Reply:
fixed